### PR TITLE
Removed audio shortcut from toolbar

### PR DESCRIPTION
**Pull request by aserna3**
In reference to https://code.google.com/p/dolphin-emu/issues/detail?id=8844

Removed the audio shortcut in the toolbar to clean up legacy design and stay consistent with current configuration management.

### CODE REVIEW

**Comment by JosJuice** [comment]
Getting rid of the audio button sounds like a good idea, but shouldn't `IDM_CONFIG_AUDIO` and `m_Bitmaps[Toolbar_ConfigAudio]` be removed entirely?

**Comment by aserna3** [comment]
I believe `m_Bitmaps[Toolbar_ConfigAudio]` can, but `IDM_CONFIG_AUDIO` has references in the menus, which AFAIK we want to keep.

**Comment by JosJuice** [comment]
MaJoR's arguments for removing Audio from the toolbar apply to removing it from the menu too.

**Comment by aserna3** [comment]
Hmm, I'm not against that. Sounds like a good idea honestly. I'll do that.

**Comment by lioncash** [comment]
I disagree with removing it from the actual menu, to be honest.

**Comment by aserna3** [comment]
Why not? I personally don't have any strong feelings keeping or removing it from the menu. I feel like it's a bit silly to have a menu option to a part of the config dialog but no menu options for the other parts, a bit inconsistent. However, the audio system is something that occasionally needs tweaking. So having a menu option would be nice.

File: Source/Core/DolphinWX/FrameTools.cpp [hunk 1]
```diff
@@ -596,7 +595,6 @@ void CFrame::InitBitmaps()
 	m_Bitmaps[Toolbar_Pause      ].LoadFile(dir + "pause.png",      wxBITMAP_TYPE_PNG);
 	m_Bitmaps[Toolbar_ConfigMain ].LoadFile(dir + "config.png",     wxBITMAP_TYPE_PNG);
 	m_Bitmaps[Toolbar_ConfigGFX  ].LoadFile(dir + "graphics.png",   wxBITMAP_TYPE_PNG);
-	m_Bitmaps[Toolbar_ConfigAudio].LoadFile(dir + "audio.png",      wxBITMAP_TYPE_PNG);
```
Comment (degasus) [inline Source/Core/DolphinWX/FrameTools.cpp]: Shall we also remove this icon?

Comment (aserna3) [inline Source/Core/DolphinWX/FrameTools.cpp]: Yeah, we should. I'll do that. I'll update this PR once we come to an agreement on whether we should keep the audio shortcut in menu.

**Comment by Rukario** [comment]
I kind of agree with @lioncash.

Otherwise the Graphics options should be merged into main config window, remove Browse icon because "Path" tab got it, etc..

It will cause more confusion to the users.

If we're going for the opposite; Audio should be separated, but it will cause more hassle to the users having to close the window and open another config windows just to config others.

I believe it's actually OK to have up to two buttons that lead users to the same menu, if such button made sense to the main UI.

**Comment by MaJoRoesch** [comment]
JosJuice: the menu is a different class of UI from the toolbar. The menu bar is meant to be the main interaction for users - it has excellent density and can show lots of options easily and concisely. The toolbar however has very poor density but high visibility and ease of access, and so it is meant to act as an overflow from that for only commonly used items.

Following that, removing a little used item from the toolbar and leaving it in the menu bar makes sense, I think.

**Comment by aserna3** [comment]
Alright, as of the latest commit, this PR just removes the toolbar button, and audio.png (not needed anymore)

Any objections?

**Comment by JosJuice** [comment]
I don't care too much about leaving the menu option in there, so LGTM.

**Comment by Rukario** [comment]
Well, this will make menus in Options and toolbar less symmetric. I'd say remove them both or leave them be.

**Comment by MaJoRoesch** [comment]
They were never meant to be symmetrical, they serve entirely different roles.

**Comment by Rukario** [comment]
You're right. I'm sure it's coincide. I'm now in favor of removing them both after reading your comment here and your issue report at Google Code.

**Comment by GitToTheHub** [comment]
Senseless this pull. Who disturbs the extra audio-icon in the toolbar? Why now an extra click?

**Comment by dragonbane0** [comment]
I used the shortcut a lot as the audio is way too loud in certain games for me, so accessing the slider quickly was nice. I see no reason to remove it tbh, but ok I guess.

**Comment by GitToTheHub** [comment]
Graphic and audio are things which belongs together and both are settings which you want to configure. The graphic settings are only detached from the settings dialog, because it has much tabs to configure. The sound settings have only one tab to configure, so its still attached in the settings window. If it would have more options, it would be detached as the graphics settings too. So it would make sense to show it in the toolbar. You could detach it from the settings menu like the graphic settings for consistency ;)

**Pull request merged**